packages/autorest.typescript/src/typescriptGenerator.ts

```diff
@@ -144,7 +144,7 @@ export async function generateTypeScriptLibrary(
 
     // Format the contents if necessary
     if (isJson || isSourceCode) {
-      fileContents = prettier.format(
+      fileContents = await prettier.format(
```

Comment:
Thank you so much for the fix! Does that mean all generated codes are not format successfully?

Reply:
yes, v3 makes the method async.